nixos-modules/microvm/options.nix

```diff
@@ -148,8 +148,7 @@ in
           };
         });
       default = [];
-      example = lib.literalExpression
-        ''
+      example = lib.literalExpression /* nix */ ''
```

Comment:
I like removing the linebreak here but what about the `/* nix */`? Not gonna enforce a `nixfmt` in this project! :laughing:

Reply:
It's tree sitter magic that allows tree sitter to inject a language in the string.